.zenodo.json

```
@@ -155,19 +160,29 @@
       },
       {
         "affiliation": "Lawrence Berkeley National Laboratory",
-        "name": "Sandberg, Ryan T.",
+        "name": "Sandberg, Ryan Thor",
         "orcid": "0000-0001-7680-8733"
       },
       {
         "affiliation": "Modern Electron",
         "name": "Scherpelz, Peter",
         "orcid": "0000-0001-8185-3387"
       },
+      {
+        "affiliation": "University of Rochester, Laboratory for Laser Energetics",
+        "name": "Weichman, Kale",
```

- No comma in affiliation?
- Do you prefer to include a middle name in scientific publications, @kale-j?
```suggestion
        "affiliation": "University of Rochester Laboratory for Laser Energetics",
        "name": "Weichman, Kale J...",
```

---

The affiliation should be "Laboratory for Laser Energetics, University of Rochester" to match the style you have for other contributors
I don't use my middle name in scientific publications, so please take out the initial

---

Thank you! :pray: